packages/grafana-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -78,6 +78,10 @@ function getStyles(theme: GrafanaTheme2) {
         pointer-events: none;
       }
 
+      &[data-popper-interactive='true'] {
+        overflow-x: scroll;
```

Comment:
```suggestion
        overflow: auto;
```

Great work 👏🏾 , I think we could prevent showing the scroll when we don't need it :)